### PR TITLE
feat(ui): update input field

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

update input field using latest design

## Linear Issue Reference

LOG-1763

## Testing

UT case added
test locally
image

### CODE REVIEW

File: packages/ui/src/components/Input/index.module.scss [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
 
-  &:disabled {
-    background: var(--color-control-background-disabled);
-    color: var(--color-secondary);
+  &:focus {
+    border: _.border(var(--color-control-focus));
   }
 }
+
+.clearBtn {
```
Comment (gao-sun) [inline packages/ui/src/components/Input/index.module.scss:27]: would prefer `clearButton` over abbr.

File: packages/ui/src/components/Input/index.module.scss [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+  position: absolute;
+  right: 0;
+  top: 0;
+  transform: translate3d(-50%, 50%, 0);
```
Comment (gao-sun) [inline packages/ui/src/components/Input/index.module.scss:31]:
```suggestion
  transform: translate(-50%, 50%);
```

File: packages/ui/src/components/Input/index.module.scss [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
-  &:disabled {
-    background: var(--color-control-background-disabled);
-    color: var(--color-secondary);
+  &:focus {
+    border: _.border(var(--color-control-focus));
   }
 }
+
+.clearBtn {
+  position: absolute;
```
Comment (gao-sun) [inline packages/ui/src/components/Input/index.module.scss:28]: do we need to use absolute positioning? how about flexbox?

**Review by gao-sun — COMMENTED** [review]
don't know if it's my mis-seeing, looks like the button is not vertical centered

File: packages/ui/src/components/Input/index.module.scss (hunk 3 above)
Comment (simeng-li) [reply packages/ui/src/components/Input/index.module.scss:33]: have to keep the fullwidth styling of the input field. As browsers autocomplete apply styling change on input field only. 

i.e 
image

Comment (gao-sun) [reply packages/ui/src/components/Input/index.module.scss:33]: need to double check, it has 1~2px height difference when focus / blur

File: packages/ui/src/components/Input/index.module.scss [hunk 4]
```diff
@@ -1,23 +1,38 @@
...
+.error:focus {
+  border: _.border(var(--color-error));
+}
+
+.clearButton {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  transform: translate(-50%, -50%);
+  fill: var(--color-neutral-70);
```
Comment (demonzoo) [inline packages/ui/src/components/Input/index.module.scss:37]: should use bottom: 50%, then transform 50% on Y axis.
```css
right: 0;
bottom: 50%;
transform: translate(-50%, 50%);
```

File: packages/ui/src/components/Input/index.module.scss (hunk 3 above)
Comment (simeng-li) [reply packages/ui/src/components/Input/index.module.scss:33]: thx to @demonzoo  point out the css bug

File: packages/ui/src/components/Input/index.module.scss (hunk 4 above)
Comment (simeng-li) [reply packages/ui/src/components/Input/index.module.scss:37]: updated thx

File: packages/ui/src/components/AppContent/index.module.scss [hunk 5]
```diff
@@ -48,37 +50,48 @@
...
+  --font-title: 600 32px/40px #{$font-family};
+  --font-heading-2: 400 18px/21px #{$font-family};
+  --font-heading-2-bold: 600 18px/21px #{$font-family};
+  --font-control: 400 18px/21px #{$font-family};
 
   /* ===== Legacy Styling ===== */
   --font-headline: 600 40px/56px #{$font-family};
   --font-heading-1: 600 28px/39px #{$font-family};
-  --font-heading-2: 600 20px/28px #{$font-family};
   --font-heading-3: 600 16px/22.4px #{$font-family};
```
Comment (demonzoo) [inline packages/ui/src/components/AppContent/index.module.scss:94]: Line height should avoid odd number and decimal values

File: packages/ui/src/components/Input/index.tsx [hunk 6]
```diff
@@ -22,21 +24,32 @@ const Input = ({
...
+        type={type}
+        value={value}
+        autoComplete={autoComplete}
+        onChange={({ target: { value } }) => {
+          onChange(value);
+        }}
+      />
+      {value && (
+        <CloseIcon
+          className={classNames(styles.clearButton)}
```
Comment (demonzoo) [inline packages/ui/src/components/Input/index.tsx:46]: `classNames` is not needed here

**Review by demonzoo — APPROVED** [review]

File: packages/ui/src/components/AppContent/index.module.scss (hunk 5 above)
Comment (simeng-li) [reply packages/ui/src/components/AppContent/index.module.scss:94]: will resolve in the future pr, once aligned

File: packages/ui/src/components/Input/index.tsx (hunk 6 above)
Comment (simeng-li) [reply packages/ui/src/components/Input/index.tsx:46]: yep will update in the next PR

**Pull request merged**